ansible/roles/cuda/tasks/main.yml

```diff
@@ -24,22 +24,13 @@
   failed_when: false
   register: _cuda_driver_module_enabled
 
-- name: List nvidia driver dnf module stream versions
-  shell:
-    cmd: dnf module list nvidia-driver | grep -oP "\d+-dkms" | sort -V
-  # Output of interest from command is something like (some whitespace removed):
-  # "nvidia-driver 418-dkms   default [d], fm, ks    Nvidia driver for 418-dkms branch "
-  changed_when: false
-  register: _cuda_driver_module_streams
-  when: "'No matching Modules to list' in _cuda_driver_module_enabled.stderr"
-
 - name: Enable nvidia driver module
-  ansible.builtin.command: "dnf module enable -y nvidia-driver:{{ _cuda_driver_module_streams.stdout_lines | last }}"
+  ansible.builtin.command: "dnf module enable -y nvidia-driver:open-dkms"
```

Comment:
Seems like the [way forward](https://developer.nvidia.com/blog/nvidia-transitions-fully-towards-open-source-gpu-kernel-modules/#supported_gpus) for new cards, but will we have issues on older ones?

Reply:
it appears to either be open + cuda 12.6 or proprietary +  cuda 12.5, at least until they fix whatever the packaging issue is for the latter combo.

I don't think we have a strict support matrix TBH, the target client system has  A40 so this is OK, and as you say appears to be recommended.

So, I think this is fine.

Reply:
I've updated the PR comment re this too, had meant to do that.